lte/gateway/c/core/oai/tasks/service303/service303_task.c

```diff
@@ -81,10 +81,14 @@ static int handle_service_message(zloop_t* loop, zsock_t* reader, void* arg) {
     case APPLICATION_UNHEALTHY_MSG: {
       service303_set_application_health(APP_UNHEALTHY);
     } break;
-    case APPLICATION_STATS_MSG: {
-      service303_mme_statistics_read(
-          &received_message_p->ittiMsg.application_mme_stats_msg);
-    }
+    case APPLICATION_MME_APP_STATS_MSG: {
+      service303_mme_app_statistics_read(
+          &received_message_p->ittiMsg.application_mme_app_stats_msg);
+    } break;
```

Comment:
@ulaskozat , @themarwhal : Oversight on missing the break here, so the first sample gets recorded correctly but subsequent samples don't as the thread exists. Apologies.